domain/model/modelmigration/export.go

```diff
@@ -61,8 +60,8 @@ func (e *exportOperation) Setup(scope modelmigration.Scope) error {
 }
 
 // Execute the export and sets the environ version of the model.
-func (e *exportOperation) Execute(ctx context.Context, model description.Model) error {
-	modelUUID := coremodel.UUID(model.Tag().Id())
+func (e *exportOperation) Execute(ctx context.Context, m description.Model) error {
```

Comment:
Does the linter tell you when you're shadowing the name of a package now? If so, nice.

Reply:
It does not I don't think. This particular case from a compiler error!